src/theme/fonts.ts

```diff
@@ -0,0 +1,78 @@
+export const fontFamily = {
```

Comment:
don't these need to get added to to `useTheme`?

Reply:
oh wait ya, I need to do the text style differently -- ignore this PR for now :)